### PR TITLE
Add `file-code-2` icon

### PR DESCRIPTION
**Pull request by danielbayley**
Related to #1068.

### CODE REVIEW

File: icons/file-code-2.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M14.5 2H6c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V7.5L14.5 2z" />
```
Comment (ericfennis) [inline icons/file-code-2.svg:12]: This path should be the same as the other file icons.
```suggestion
 <path d="M14.5 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V7.5L14.5 2z" />
```

**Review by ericfennis — CHANGES_REQUESTED** [review]
Nice icon, left some code suggestions.

**Comment by karsa-mistmere** [comment]
Just an idea: maybe if we add this, let's rename the current `file-code` to `file-code-2` and add this one as `file-code`, so that the layouts are consistent (no suffix => contents inside, `-2` suffix => contents to the side).

**Comment by danielbayley** [comment]
> Just an idea: maybe if we add this, let's rename the current `file-code` to `file-code-2` and add this one as `file-code`, so that the layouts are consistent (no suffix => contents inside, `-2` suffix => contents to the side).

Yeah I noticed this inconsistency also… Would this require adding an alias in this particular case?

**Comment by karsa-mistmere** [comment]
No, we can't add an alias for a name that will already taken.

File: icons/file-code-2.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "script",
+    "document"
+  ],
+  "categories": [
+    "files",
+    "coding"
```
Comment (karsa-mistmere) [inline icons/file-code-2.json:9]:
```suggestion
    "development"
```

File: icons/file-code.json [hunk 3]
```diff
@@ -2,7 +2,12 @@
   "$schema": "../icon.schema.json",
   "tags": [
     "script",
-    "document"
+    "document",
+    "gist",
+    "html",
+    "xml",
+    "property list",
+    "plist"
```
Comment (karsa-mistmere) [inline icons/file-code.json:10]: Why not add all these tags to the other version?

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

**Pull request merged**